PanoptesNetClient/PanoptesNetClient/ApiClient.cs

```diff
@@ -6,19 +6,28 @@
 
 namespace PanoptesNetClient
 {
-    public class Project
-    {
-        public string Id { get; set; }
-    }
-
     public class ApiClient
     {
         static HttpClient client = new HttpClient();
```

Comment:
I think you this should be `private` instead of `static`. Because this class is a singleton it already assures that it will only initialize the client once.